README.md

```diff
@@ -1,3 +1,26 @@
 # elm-css-normalize
 
-A port of normalize.css for elm-css
+A port of normalize.css for elm-css.
+
+
+## Usage
```

Comment:
I'd prefer Usage docs be restricted to examples in the Elm file's comments/docs. It's a typical Elm pattern to have a minimal README and put everything in the Elm files.

This is mostly so it shows up correctly on the Elm Package manager